arches/settings_utils.py

```diff
@@ -121,38 +129,63 @@ def build_templates_config(
     ]
 
 
-def transmit_webpack_django_config(
-    root_dir,
-    app_root,
-    static_url,
-    public_server_address,
-    webpack_development_server_port,
-    arches_applications=None,
-):
-    arches_applications_paths = {}
+def inject_arches_applications_directories():
```

Comment:
An issue I have with this is that it's obfuscating how `STATICFILES_DIRS` and `TEMPLATES` gets built. Imagine I'm a new Arches user and I want to know how my Arches-application templates are getting rendered. The function called in `settings.py` has no reference to it and without combing the rest of `settings_utils` I'm left not knowing how the thing is made.

This could be a signposting issue. But preferably I'd like the thing to get built in all the same place.

Reply:
Yeah if this able to be removed entirely in favor of that setting change, I'd say it's worth being folded into this ticket.